Training24/TestComplex.c

```diff
@@ -0,0 +1,59 @@
+// ------------------------------------------------------------------------------------
+// Training ~ A training program for new joiners at Metamation, Batch - July 2024.
+// Copyright (c) Metamation India.
+// Dhaarani Mohanasundaram
+// -------------------------------------
+// TestComplex.c
+// Program on Test1.2 branch.
+// ------------------------------------------------------------------------------------
+#include <stdio.h>
+#include <math.h>
+#include <stdbool.h>
+#include "Complex.h"
+
+#define RESET "\033[0m"
+#define GREEN "\033[32m"
+#define RED "\033[31m"
+
+/// <summary>To check if both the imaginary and real part of two complex number are equal.</summary>
+bool Equal (ComplexNumber c1, ComplexNumber c2) {
+   return (c1.Real == c2.Real) && (c1.Img == c2.Img);
+}
+
+void RunTestCases () {   // TestCase Array as  C1 {r1, img1}, C2 {r2, img2}, C3 {r3, img3} 
+   ComplexNumber input[] = { {5, 4}, {6, -5}, {2, 3} },
+      // Operations for add,sub,mul as { C1 & C2} ,{ C2 & C3 },{ C3 & C1 }
+      expAdd[] = { {11, -1}, {8, -2}, {7, 7} },
+      expSub[] = { {-1, 9}, {4, -8}, {-3, -1} },
+      expMul[] = { {50, -1}, {27, 8}, {-2, 23} },
+      expConj[] = { {5, -4}, {6, 5}, {2, -3} };   // C1, C2, C3
+   float expMod[] = { 6.4031, 7.8102, 3.6056 };
+   for (int i = 0; i < 3; i++) {
```

Comment:
Good use of the "for" loop.
Say you want to add more test cases. is there any way you can avoid using the magic number 3 to control the "for" loop?